File: GithubCopilot_BP/CWE-190/190_2c_BP.py
import secrets

# Generate random integers between 10 and 1000
num1 = secrets.randbelow(991) + 10
num2 = secrets.randbelow(991) + 10

# Calculate the absolute difference
difference = abs(num1 - num2)

# Print the results
print(f"The difference between {num1} and {num2} is {difference}.")

File: GitHubCopilot_BP_Java/CWE-190/190_2c_BP.java
import java.security.SecureRandom;

public class RandomDifference {
    public static void main(String[] args) {
        SecureRandom random = new SecureRandom();

        // Generate random integers between 10 and 1000
        int num1 = 10 + random.nextInt(991);
        int num2 = 10 + random.nextInt(991);

        // Calculate the absolute difference
        int difference = Math.abs(num1 - num2);

        // Print the results
        System.out.printf("First number: %d%nSecond number: %d%nDifference: %d%n", num1, num2, difference);
    }
}